.wordlist.txt

```diff
@@ -214,3 +214,10 @@ G4DN
 DN
 OCID
 AArch
+firewalling
+ec
+authorize
+authorized
+pem
+https
+Signon
```

Comment:
These might need updating. We use UK spelling, so use 'authorise' with an 's'. I haven't checked where all the others are used, but their case might have to be changed to maintain consistency and e.g. Signon won't be needed here since it is now part of a :guilabel: (comment below)